src/core/server/queue/tasks/mailer/templates/update-username.html

```diff
@@ -0,0 +1,11 @@
+{% extends "layouts/user-notification.html" %} {% block content %} Hello
```

Comment:
@wyattjoh  showed me that you can use the Nunjucks extension with VSCode and it will stop this from being garbled every time you save.

Link to extension: https://github.com/ronnidc/vscode-nunjucks